stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/channel/list/adapter/viewholder/ChannelListItemViewHolder.kt

```diff
@@ -19,18 +19,9 @@ import io.getstream.chat.android.ui.utils.extensions.getLastMessage
 import io.getstream.chat.android.ui.utils.extensions.getLastMessageTime
 import io.getstream.chat.android.ui.utils.extensions.getReadStatusDrawable
 import io.getstream.chat.android.ui.utils.extensions.getUsers
-import java.text.SimpleDateFormat
-import java.util.Locale
 
 public class ChannelListItemViewHolder(itemView: View) : BaseChannelListItemViewHolder(itemView) {
 
-    public companion object {
-        @SuppressLint("ConstantLocale")
-        private val DEFAULT_LOCALE: Locale = Locale.getDefault()
-
-        private val TIME_FORMAT = SimpleDateFormat("hh:mm", DEFAULT_LOCALE)
```

Comment:
This was actually broken, as `hh` is the format for 12-hour time, so everything past 12:00 had 12 hours subtracted from it